pkg/oc/bootstrap/docker/openshift/admin.go

```diff
@@ -90,26 +90,14 @@ func (h *Helper) InstallRegistry(dockerClient dockerhelper.Interface, ocImage st
 	if rc != 0 {
 		return errors.NewError("could not run %q: rc==%v", componentName, rc)
 	}
-
-	svc, err := kubeClient.Core().Services(DefaultNamespace).Get(SvcDockerRegistry, metav1.GetOptions{})
-	if err == nil {
```

Comment:
just fix this if statement and leave the service re-write bit in place.  Otherwise noproxy is likely to bite us and that's trouble we dont' need yet.

Reply:
i thought that you can use a CIDR form for noproxy, so `NO_PROXY=172.30.0.0/16` will cover the registry and we don't have to set the cluster ip.